### PR TITLE
Release notes for WebAuthn idless

### LINKED ISSUE #1437: Release notes for WebAuthn idless
### Description

WebAuthn idless was community contribution already added into Keycloak "main" . This task is only about adding release notes for it.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by mposolda**
Closes #1437

### CODE REVIEW

**Comment by mposolda** [comment]
@andymunro Do you please have a chance to review?

File: release_notes/topics/18_0_0.adoc [hunk 1]
```diff
@@ -11,3 +11,10 @@ Recovery Codes as another way to do two-factor authentication is now available a
 For more details, take a look at link:{adminguide_link}#_step-up-flow[{adminguide_name}].
 
 Thanks to https://github.com/CorneliaLahnsteiner[Cornelia Lahnsteiner] and https://github.com/romge[Georg Romstorfer] for the contribution.
+
+== WebAuthn improvements
+
+{project_name} now supports WebAuthn id-less authentication. This allows that WebAuthn Security Key will identify the user during authentication as long as the
```
Comment (andymunro) [inline release_notes/topics/18_0_0.adoc]: Try to always follow "this" with a noun, so it's clear what "this" is referring to.  This feature or this change..

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good overall, @mposolda.

File: release_notes/topics/18_0_0.adoc (hunk 1 above)
Comment (mposolda) [reply release_notes/topics/18_0_0.adoc]: Thanks for the suggestion! It is my favourite mistake :-( I will try to pay more attention next time.

I changed it to "This feature" as you suggested.

**Comment by mposolda** [comment]
@mabartos Do you please have a chance to review this and/or eventually propose any changes related to WebAuthn?

**Review by andymunro — DISMISSED** [review]

File: release_notes/topics/18_0_0.adoc [hunk 2]
```diff
@@ -11,3 +11,10 @@ Recovery Codes as another way to do two-factor authentication is now available a
 For more details, take a look at link:{adminguide_link}#_step-up-flow[{adminguide_name}].
 
 Thanks to https://github.com/CorneliaLahnsteiner[Cornelia Lahnsteiner] and https://github.com/romge[Georg Romstorfer] for the contribution.
+
+== WebAuthn improvements
+
+{project_name} now supports WebAuthn id-less authentication. This feature allows that WebAuthn Security Key will identify the user during authentication as long as the
+security key supports Resident Keys. Thanks to https://github.com/vanrar68[Joaquim Fellmann] for the contribution.
```
Comment (mabartos) [inline release_notes/topics/18_0_0.adoc]: Probably, it'd be nice to mention a link to the Loginless WebAuthn section, when customers could check it and find out how's it cool?

**Review by mabartos — COMMENTED** [review]
Thanks, @mposolda. I think it's sufficient and there's no need to add something else. Just minor thing below.

File: release_notes/topics/18_0_0.adoc (hunk 2 above)
Comment (mposolda) [reply release_notes/topics/18_0_0.adoc]: @mabartos Good point! I've added the link to the documentation.

**Review by mabartos — APPROVED** [review]

**Comment by mposolda** [comment]
@mabartos Thanks for the review!

**Pull request merged**